services/storage/src/simcore_service_storage/api/_worker_tasks/_simcore_s3.py

```diff
@@ -53,3 +60,44 @@ async def deep_copy_files_from_project(
             )
 
     return body.destination
+
+
+async def data_export(
+    task: Task,
+    task_id: TaskID,
+    *,
+    user_id: UserID,
+    paths_to_export: list[S3ObjectKey],
+) -> StorageFileID:
+    _logger.info("Exporting (for user='%s') selection: %s", user_id, paths_to_export)
```

Comment:
use `with log_context` instead. and then I am not sure the other log you have deeper still make sense if this is already here as well. Let's not polute the log too much.